Android/app/src/main/java/com/vigbag/android/MainActivity.kt

```diff
@@ -2,15 +2,15 @@ package com.vigbag.android
 
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
-import androidx.fragment.app.Fragment
 import com.vigbag.android.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var binding:ActivityMainBinding
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        setTheme(R.style.Theme_VigyBag)
```

Comment:
Please use `installSplashScreen()` function by importing `implementation("androidx.core:core-splashscreen:1.1.0")` instead of setTheme here.

Reply:
Actually, remove all of the code changes in this file. Essy is already assigned with splashscreen issue.